CLAUDE.md

```
@@ -4,91 +4,159 @@ This file provides guidance to Claude Code (claude.ai/code) when working with co
 
 ### Monorepo Structure
-- **frontend/** - React application (Vite, TypeScript, TailwindCSS, Flowbite)
-- **backend/** - PocketBase container (Alpine Linux, port 8090)
+- **mobile/** - Expo React Native app (active development)
+- **frontend/** - Legacy React/Vite web app (deprecated)
+- **backend/** - PocketBase container (deprecated, replaced by local TinyBase)
 - **docs/** - Jekyll documentation site for GitHub Pages
 
-### Frontend Organization
+### Mobile App Organization
 ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)